updates/1.126/221108_journalFeedback.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just a reminder to double-check database version number before merging into devel (currently OK).